CommonMark/Syntax/Inline.cs

```diff
@@ -205,5 +206,14 @@ public Inline LastSibling
                 return x;
             }
         }
+
+        /// <summary>
+        /// Gets the delimiter character for this inline element.
+        /// </summary>
+        public char DelimiterCharacter
+        {
+            get;
+            private set;
```

Comment:
no, unfortunately those cannot be changed as that would be a breaking change. I do not know why anybody would want to change those values but somebody might be using that.